Update revision-notes.php

Kenshino opened the pull request:
Plugins loaded is too early - themes that register post types, or even plugins that register post type to init won't benefit from this plugin.

Set to add_action to init and priority to 99

Kenshino committed:
Update revision-notes.php

Plugins loaded is too early - themes that register post types, or even plugins that register post type to init won't benefit from this plugin.

Set to add_action to init and priority to 99

helen commented:
Oh awesome, thanks. One thing - if you could also update the name of that class method from `plugins_loaded` to `init` (or something more descriptive) that'd be lovely. I tend to match the hook and method names but they don't have to.

Kenshino committed:
Update revision-notes.php

Change function name from plugins_loaded to init to mirror point of activation
Added path for language files to use /languages as per what plugins usually do

Kenshino commented:
@helen - fixed :)
